README.md

```diff
@@ -24,6 +24,13 @@ The tool dispatches and runs trial jobs generated by tuning algorithms to search
 * Researchers and data scientists who want to implement their own AutoML algorithms and compare it with other algorithms.
 * ML Platform owners who want to support AutoML in their platform.
 
+## Related Projects
+Targeting at openness and advancing state-of-art technology, [Microsoft Research (MSR)](https://www.microsoft.com/en-us/research/group/systems-research-group-asia/) had also released few other open source projects.
+* [OpenPAI](https://github.com/Microsoft/pai) : an open source platform that provides complete AI model training and resource management capabilities, it is easy to extend and supports on-premise, cloud and hybrid environments in various scale.
+* [FrameworkController](https://github.com/Microsoft/frameworkcontroller) : an open source general-purpose Kubernetes Pod Controller that orchestrate all kinds of applications on Kubernetes by a single controller.
+* [MMdnn](https://github.com/Microsoft/MMdnn) : A comprehensive, cross-framework solution to convert, visualize and diagnose deep neural network models. The "MM" in MMdnn stands for model management and "dnn" is an acronym for deep neural network.
+We encourage researchers and students leverage these projects to accelerate the AI development and research.
+
```

Comment:
I suggest to put this part after `Tutorials`.

Reply:
@QuanluZhang  - the position is on purpose putting ahead of tutorials. Had discussed this with Mao previously when adding our reference link to PAI and MMdnn. these 3 projects are currently align the same "related project" format.

https://github.com/Microsoft/pai
https://github.com/Microsoft/mmdnn